values.yaml

```diff
@@ -21,6 +21,18 @@ coderd:
     # will be restricted to the specified client IPs. This field will be ignored if
     # the cloud provider does not support this feature.
     loadBalancerSourceRanges: []
+
+  # coderd.serviceNodePorts -- Allows manually setting static node ports for the coderd service.
+  # This is only helpful if static ports are required, and usually should be left alone.
+  # By default these are dynamically chosen.
+  serviceNodePorts:
```

Comment:
By adding these to the values file, these will be the default values applicable to all installs - is that what we want? What's the default behavior (e.g. if nodePort isn't specified)?

Reply:
The `values.yaml` is the default?
The original default is that the nodePorts are selected dynamically. I thought `values.yaml` was an example config for generating the docs.

Reply:
the values file is the defaults that will be used if you don't specify any that override them, it's included in the Helm chart package (so if you do: `helm install . coder` then it'll use the `values.yaml` file unmodified). otherwise, if you specify --set, then it'll merge the default values file with the args you pass

Reply:
TIL. So we can't document a field without explicitly setting a default?
I wonder if I can set it to `null`?

I prefer the default to be unset.

Reply:
I could also make the value "0" effectively be "unset"

Reply:
Yeah I think null should work

Reply:
You can test by running `make lint/kubernetes`, this will generate a bunch of YAML files somewhere that you can examine

Reply:
`null` works